client/lib/analytics/ad-tracking/woo/is-wooexpress-upgrade.ts

```diff
@@ -0,0 +1,41 @@
+import { PLAN_ECOMMERCE_TRIAL_MONTHLY, isEcommerce } from '@automattic/calypso-products';
+import { SiteDetails } from '@automattic/data-stores/src';
```

Comment:
Same comment here about importing from the package's public exports.

```suggestion
import type { SiteDetails } from '@automattic/data-stores';
```